runtime/jcl/exports.cmake

```diff
@@ -629,6 +629,8 @@ if(J9VM_OPT_OPENJDK_METHODHANDLE)
 		Java_java_lang_invoke_MethodHandleNatives_clearCallSiteContext
 		Java_java_lang_invoke_MethodHandleNatives_getNamedCon
 		Java_java_lang_invoke_MethodHandleNatives_registerNatives
+		Java_java_lang_invoke_MethodHandle_invoke
```

Comment:
you can do a similar change here (like with UMA) and just define these in the first export block.

Also, you can remove the exports in line 442,443